### ISSUE TITLE
VarSet: add property dialog needs better accessibility, cannot use keyboard to navigate through fields

### ISSUE THREAD

**Issue by furgo16** [FreeCAD/FreeCAD#17067]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Steps to reproduce:

1. Start FreeCAD; create new document
2. Create a new VarSet, as described in https://wiki.freecad.org/Std_VarSet
3. On the Add property dialog, enter a name on the first field (`Name`)
4. Press the <kbd>Tab</kbd> key on the keyboard to navigate to the next field (`Group`)
4. Press the <kbd>Tab</kbd> key on the keyboard to navigate to the next field (`Type`)
5. **Expected**: focus is on the _next_ field (`Type`)
5. **Actual**: focus is on the _previous_ field (`Name`)

The effect is that repeatedly pressing the <kbd>Tab</kbd> key simply alternates between the `Name` and `Group` field in a loop. The rest of fields cannot be accessed via keyboard alone.

It would seem that the tab order needs to be defined on that dialog.

https://github.com/user-attachments/assets/3fa2ad5f-ca7d-4805-a602-fd1c1f6cfb16

### Full version info

```shell
OS: Ubuntu Core 22 (ubuntu:GNOME/ubuntu)
Word size of FreeCAD: 64-bit
Version: 1.1.0dev.38898 (Git) Snap 1157
Build type: Release
Branch: main
Hash: 32e09d9554db26414697ed19e0189ccab4ea9029
Python 3.10.12, Qt 5.15.10, Coin 4.0.0, Vtk 7.1.1, OCC 7.7.1
Stylesheet/Theme/QtStyle: OpenLight.qss/OpenLight/Qt default
Installed mods: 
  * OpenTheme 2024.9.1
  * ezydxf
  * Ondsel-Lens 2024.9.13.02
  * Assembly4 0.50.15
```

### Subproject(s) affected?

Core

### Anything else?

1. Seeing the [number of issues](https://github.com/orgs/FreeCAD/projects/30) this dialog has, I wonder if it would not be best to start using the original "add property" dialog and simply adding the extended functionality to it (https://github.com/FreeCAD/FreeCAD/issues/14633).
1. When the `Type` field is clicked, the tab order works correctly from there (from `Type` to `OK` and back only).
1. I could not figure out how to fix the tab order, but while looking at it, I also noticed that the dialog title ("Add a property to ") needs to be internationalized, as it looks like it's not translatable. I might file a follow up issue.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by furgo16** [comment]
Related issue: https://github.com/FreeCAD/FreeCAD/issues/15720 (not quite a duplicate, as it shows VarSet as an example, but it refers to other input dialogs)
